src/porepy/numerics/ad/grid_operators.py

```diff
@@ -87,12 +87,13 @@ def cell_restriction(self, grids: List[pp.Grid]) -> Matrix:
             if len(grids) > 0:
                 # A key error will be raised if a grid in g is not known to
                 # self._cell_projection
+                # IMPLEMENTATION NOTE: Use csc format, since the number of rows can
```

Comment:
This comment and those below: The note has been copy-pasted and is not consistent with the code in the cases where you have used .tocsr()

Reply:
This is by purpose, I wrote explanation below.

Reply:
I don't get it

Reply:
Sorry, my response was completely wrong. Fixed it.